packages/relay/src/services/singletons/TransactionManager.ts

```diff
@@ -163,7 +164,7 @@ export class TransactionManager {
                     (log: ethers.utils.LogDescription | null) => log !== null
                 )
         }
-        return parsedLogs ?? null
+        return { txHash: hash, logs: parsedLogs }
```

Comment:
Do you know what does this parseLogs use for ?

Reply:
It's used to get logs from on-chain events

https://github.com/social-tw/social-tw-website/blob/3412f14af604c451ec26104d7ae34ad9da5c2a20/packages/relay/src/services/PostService.ts#L246